drakrun/drakrun/systemd/drak-vnc@.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=drak-vnc service
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/etc/drakrun/scripts/drak-vnc %i
+User=root
+Group=root
```

Comment:
I'd like to have less services running as root. This script doesn't require any special privileges so we can run it as nobody.

Reply:
+1